addons/stock/models/stock_lot.py

```diff
@@ -149,12 +151,15 @@ def _compute_delivery_ids(self):
     def _compute_last_delivery_partner_id(self):
         serial_products = self.filtered(lambda l: l.product_id.tracking == 'serial')
         delivery_ids_by_lot = serial_products._find_delivery_ids_by_lot()
-        (self - serial_products).last_delivery_partner_id = False
+        (self - serial_products).last_delivery_partner_id = (self - serial_products).last_delivery_date = (self - serial_products).quantity = False
```

Comment:
I would use `.write` for common field write

Reply:
Done.